Pipfile

```diff
@@ -24,6 +24,8 @@ requests = "==2.31.0"
 slack-sdk = "==3.27.1"
 toml = "==0.10.2"
 datetime = "*"
+pytest = "*"
```

Comment:
Why is this here? I can't see it being used anywhere?

Reply:
Removed